manifests/p/PKI/NCALayer/1.3/PKI.NCALayer.installer.yaml

```diff
@@ -1,24 +1,25 @@
-# Created with YamlCreate.ps1 v2.1.0 $debug=NVS0.5-1-19041-1320
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
 
 PackageIdentifier: PKI.NCALayer
 PackageVersion: "1.3"
+Platform:
+- Windows.Desktop
 InstallerLocale: en-US
 MinimumOSVersion: 10.0.0.0
 InstallerType: inno
 InstallModes:
 - silent
 - silentWithProgress
 UpgradeBehavior: install
-ReleaseDate: 2022-03-13
+ReleaseDate: 2024-01-08
 Installers:
 - Architecture: x64
   InstallerUrl: https://ncl.pki.gov.kz/images/NCALayer/NCALayerInstall_x64.exe
-  InstallerSha256: 8A5D1818F5B2E2B376F9DD950EDF495F1394467FE7A79D1A5040B603D62D764D
+  InstallerSha256: 4BDABDFD13FCEA5C52A8415A61FBF4888558206539175E2F2F5E38F9326EE654
   ProductCode: '{2E9A81FF-B19F-4F64-95B0-E688E09F7D7C}_is1'
 - Architecture: x86
   InstallerUrl: https://ncl.pki.gov.kz/images/NCALayer/NCALayerInstall_x86.exe
-  InstallerSha256: 6662D35AD82C40DD04773C85AEB5FA729893A16E4D2B53CFA4841D78E9948480
+  InstallerSha256: CCDDB42CA18234DF422730CE61EDFC645E3B5962A21F623DCAEDEACDCE1A37D9
```

Comment:
Usually when the SHA256 changes it means that a new version is available; This may need to be a new PackageVersion?

Reply:
The version of the application has not changed, all the same 1.3